service/authorization/authorization.go

```diff
@@ -2,13 +2,15 @@ package authorization
 
 import (
 	"context"
+	"encoding/json"
 	"errors"
 	"fmt"
 	"log/slog"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/golang-jwt/jwt"
```

Comment:
Can you switch any usage of `golang-jwt` to the `lestrrat-go`.

Reply:
switched